pkgs/applications/kde/mbox-importer.nix

```diff
@@ -15,6 +15,7 @@ mkDerivation {
     akonadi akonadi-search kconfig kservice kio mailcommon mailimporter messagelib
   ];
   preHook = ''
```

Comment:
@xeji I must have left this in here by accident. Could you just remove the `preHook` entirely?

Reply:
done